_data/simple-icons.json

```diff
@@ -14835,6 +14835,17 @@
             "hex": "000000",
             "source": "https://society6.com"
         },
+        {
+            "title": "Socket.dev",
+            "hex": "C93CD7",
+            "source": "https://socket.dev",
```

Comment:
Where did you get the SVG source file? I only see the PNG logo on the homepage.

We need the SVG source file for reviewing purposes.

Reply:
I updated the source field to reflect the official socketDev GitHub repository which I found it in:
https://github.com/SocketDev/vscode-socket-security/blob/master/logo-red.svg